README.md

```diff
@@ -223,23 +203,6 @@ Count number of events in each queue:
 jq 'map_values(map(keys[0]))' queue_dump.json
 ```
 
-### Diagnostic console
-
-If the configuration option `console.enabled` is set to `true`, a unix socket named `debug.socket` by default can be found next to the configuration while the node is running. It can be connected to by tools like `socat`:
-
-```sh
-$ socat - unix:/path/to/debug.socket
-```
-
-Entering `help` will show available commands. The interface can also be scripted, it may be helpful to change some of the per-connection settings in this case:
-
-```sh
-echo -e 'set -q true -o bincode\nexample-command' | socat - unix-client:/path/to/debug.socket > output.dump
-```
-
-This calls `example-command` on the console, but disables the command confirmation (`-q true`) and changes the output format to bincode (`-o bincode`) before doing so.
-
-
```

Comment:
Not sure why changes to this file are part of the PR.

Reply:
It may have been introduced unintentionally as part of resolving merge conflicts